vavr/src/main/java/io/vavr/control/Either.java

```diff
@@ -414,6 +414,44 @@ default void orElseRun(Consumer<? super L> action) {
         return isLeft() || predicate.test(get()) ? Option.some(this) : Option.none();
     }
 
+    /**
+     * Filters this right-biased {@code Either} by testing a predicate.
+     * If the {@code Either} is a Right and the predicate doesn't match, the
+     * {@code Either} will be turned into a {@code Left} with contents computed by applying
+     * the filterVal function to the {@code Either} value.
+     * <p>
+     *
+     * <pre>{@code
+     * import static io.vavr.API.*;
+     *
+     * class Example {
```

Comment:
I would either remove the class declaration (and keep the example/body) or use a static initializer because the current code is not valid Java.